js/scripts.js

```
@@ -2,38 +2,2573 @@
+    // Kill GSAP context for proper cleanup
+    if (ctx) {
+      ctx.kill();
+    }
+
+    // reset all scroll triggers
+    let triggers = ScrollTrigger.getAll();
+    triggers.forEach( trigger => {
+		  	trigger.kill();
+    });
+
+    ClapatSlider.instances.forEach(slider => slider.off());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)